modules/installation-azure-service-principal.adoc

```diff
@@ -177,87 +173,41 @@ output. You need these values during {product-title} installation.
 +
 [source,terminal]
 ----
-$ az ad sp create-for-rbac --role Contributor --name <service_principal> <1>
+$ az ad sp create-for-rbac --role Contributor --name <service_principal> \ <1>
+  --scopes /subscriptions/53b8f551-f0fc-4bea-8cba-6d1fefd54c8a
 ----
-<1> Replace `<service_principal>` with the name to assign to the service principal.
+<1> Specify the service principal name.
 +
 .Example output
 [source,terminal]
 ----
-Changing "<service_principal>" to a valid URI of "http://<service_principal>", which is the required format used for service principal names
+Changing "<service_principal>" to a valid URI of "http://<service_principal>",
+which is the required format used for service principal names
 Retrying role assignment creation: 1/36
 Retrying role assignment creation: 2/36
 Retrying role assignment creation: 3/36
 Retrying role assignment creation: 4/36
 {
-  "appId": "8bd0d04d-0ac2-43a8-928d-705c598c6956",
+  "appId": "b22af74d-f4f7-47a9-af38-9bed02b0004c",
   "displayName": "<service_principal>",
-  "name": "http://<service_principal>",
-  "password": "ac461d78-bf4b-4387-ad16-7e32e328aec6",
-  "tenant": "6048c7e9-b2ad-488d-a54e-dc3f6be6a7ee"
+  "password": "ac461d78-bf4b-4387-ad16-7e32e328aec6", <1>
+  "tenant": "6047c7e9-b2ad-488d-a54e-dc3f6be6a7ee"
 }
 ----
```

Comment:
the output is : 
Creating 'Contributor' role assignment under scope '//subscriptions/53b8f551-f0fc-4bea-8cba-6d1fefd54c8a'
The output includes credentials that you must protect. Be sure that you do not include these credentials in your code or check the credentials into your source control. For more information, see https://aka.ms/azadsp-cli
{
  "appId": "b22af74d-....",
  "displayName": "<service_principal>",
  "password": "vSL8Q...",
  "tenant": "8049c7e9-c3de-762d-a54e-dc3f6be6a7ee"
}